src/GregClient/Requests/GetMaintainers.cs

```diff
@@ -0,0 +1,32 @@
+﻿using RestSharp;
+
+namespace Greg.Requests
+{
+    public class GetMaintainers : PackageReferenceRequest
```

Comment:
@zeusongit can you add /// comments for public members and make those which dont need to be public internal?